client/src/test/java/io/pravega/client/stream/ReaderGroupConfigTest.java

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.client.stream;
+
+import com.google.common.collect.ImmutableMap;
+import io.pravega.client.segment.impl.Segment;
+import io.pravega.client.stream.impl.CheckpointImpl;
+import io.pravega.client.stream.impl.StreamCutImpl;
+import java.util.Map;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.when;
+
+public class ReaderGroupConfigTest {
+    private static final String SCOPE = "scope";
+
+    @Test
+    public void testValidConfig() {
+        ReaderGroupConfig cfg = ReaderGroupConfig.builder()
+                .disableAutomaticCheckpoints()
+                .stream("scope/s1", getStartStreamCut("s1"))
+                .stream("scope/s2", getStartStreamCut("s2"))
+                .build();
+
+        assertEquals(-1, cfg.getAutomaticCheckpointIntervalMillis());
+        assertEquals(3000L, cfg.getGroupRefreshTimeMillis());
+        assertEquals(getStartStreamCut("s1"), cfg.getStartingStreamCuts().get(Stream.of("scope/s1")));
+        assertEquals(getStartStreamCut("s2"), cfg.getStartingStreamCuts().get(Stream.of("scope/s2")));
+    }
+
+    @Test
+    public void testValidConfig2() {
```

Comment:
Meaningful names rather numerical values, please. Same for all test cases here.

Reply:
Done, removed numerical values in test method names.